server/routes/applications.js

```diff
@@ -66,26 +66,29 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   logger.info(`GET request to view a single application: ${req.path}`);
   const id = req.params.appId;
   const env = req.params.env;
-  const viewData = routeUtils.createViewData('View application', 'view-application', req);
+  const viewData = {
```

Comment:
So the createView data also displays the email address in the header. What you'll probably find is that the email is missing from the page. You can add it in by putting

`user_profile: req.session.data.signin_info.user_profile`

in the viewData object